typos.toml

```
@@ -0,0 +1,7 @@
+files.extend-exclude = [
+  # "CHANGELOG.md",
+]
+[default.extend-words]
+# french
+projet = "projet"
+cristal = "cristal"
```

The French words "projet" and "cristal" appear in the OCaml test sample files. I don't think we care about typos in the test samples, so I would suggest:

```suggestion
files.extend-exclude = [
  "/topiary-cli/tests/samples",
]
```

Although, per my previous comment, in `Cargo.toml`, rather than here.